### ISSUE TITLE
Depositing ReScience articles in preprint archives

### ISSUE THREAD

**Issue by khinsen** [ReScience/ReScience#95]
@ReScience/editors Following @broukema's experience (see https://github.com/ReScience/ten-years/issues/2), should we add some recommendation about submission of articles to preprint archives to our instructions for authors? Depositing ReScience C papers elsewhere sounds like a good and cheap way to get more attention, both for the paper and for ReScience C.

**Comment by rougier** [comment]
I agree we could do that. Note that we're also in the process of moving to JOSS infrastructure with proper referencing from Google Scholar.

**Comment by labarba** [comment]
hey, @rougier -- what do you mean by "int he process of moving to JOSS infrastructure"? Are you cloning the whole tooling, and re-hosting under your own domain? Or is ReScience coming under The Open Journals umbrella? I believe only the latter will give you immediate Google Scholar indexing. We worked over a couple of years to get TheOJ recognized as a legit publisher by the Scholar team, with a bunch of tweaks needed under the hood for metatags and whatnot. Cloning the infra under your own domain, I think, means going back to square one for Google Scholar to index you.

**Comment by rougier** [comment]
It is not yet clear actually. @karthik told us (https://github.com/ReScience/articles/issues/11) he was in the process of generalizing JOSS and ReScience C might be part of the process. I guess we would go for the simplest option as long as we can keep the rescience.github.io domain.

**Comment by khinsen** [comment]
@rougier I wasn't that much thinking of Google Scholar, which is an issue we should tackle at the ReScience level. But preprint servers are gaining in popularity, and I see more and more scientists keeping an eye on their domain's dominant preprint server, so this is a good way for ReScience articles to get noticed.

BTW, I am not proposing that we do the submissions (on some servers that would even be impossible). I am merely proposing that we point out to authors that this could be useful to do, and perhaps provide some kind of instructions if we think that could help.
